### ISSUE TITLE
Need to adjust rgw_frontends syntax on upgrade?

### ISSUE THREAD

**Issue by smithfarm** [SUSE/DeepSea#638]
From jewel/SES4 to luminous/SES5 there was a change in the rgw_frontends syntax when specifying multiple rgw ports.

In jewel it was

> rgw_frontends = "civetweb port=80, civetweb port=443s ssl_certificate=/etc/ceph/rgw.pem"

whereas in luminous it is

> rgw_frontends = "civetweb port=80+443s ssl_certificate=/etc/ceph/rgw.pem"

Does engulf/upgrade handle this already?

**Comment by smithfarm** [comment]
@Martin-Weiss @theanalyst @kmroz @tserong @jschmid1

**Comment by tserong** [comment]
No, it just pulls in the config file as-is, it doesn't rewrite that.

**Comment by smithfarm** [comment]
Thanks, @tserong - updated title and description.

**Comment by tserong** [comment]
Also note that the import saves ceph.conf to /srv/salt/ceph/configuration/files/ceph.conf.import, and sets `configuration_init: default-import`, which makes DeepSea use the imported configuration file, rather than using DeepSea's default /srv/salt/ceph/configuration/files/ceph.conf.j2 template.

**Comment by theanalyst** [comment]
I'm not sure whether the behaviour changed between jewel & L, @kmroz the frontends permissions drop went in for jewel itself right?

**Comment by smithfarm** [comment]
@kmroz ping?
